api/src/changeRequests/validateChangeRequest.spec.ts

```diff
@@ -156,4 +158,50 @@ describe("validateChangeRequest", () => {
         const result = await validateChangeRequest(changeRequest, ["group-super-admins"], db);
         expect(result.validatedData.invalidField).toBe(undefined);
     });
+
+    it("fails validation on an invalid uploaded image document", async () => {
+        const changeRequest = {
+            id: 42,
+            doc: {
+                _id: "image-test",
+                type: "image",
+                name: "Test Image",
+                uploadData: [
+                    {
+                        fileData: Buffer.from("some invalid data"),
+                        preset: "default",
+                    },
+                ],
+            },
+        };
+
+        const result = await validateChangeRequest(changeRequest, ["group-super-admins"], db);
+
+        expect(result.validated).toBe(false);
+        expect(result.error).toBeDefined();
+    });
+
+    it("validates a valid uploaded image document", async () => {
+        const changeRequest = {
+            id: 42,
+            doc: {
+                _id: "image-test",
+                type: "image",
+                name: "Test Image",
+                uploadData: [
+                    {
+                        fileData: fs.readFileSync(
+                            path.resolve(__dirname + "/../test/" + "testImage.jpg"),
+                        ),
+                        preset: "default",
+                    },
+                ],
+            },
+        };
+
+        const result = await validateChangeRequest(changeRequest, ["group-super-admins"], db);
+
+        expect(result.validated).toBe(false);
+        expect(result.error).toBeDefined();
```

Comment:
This test is called "validates a valid document" but the actual expect says that the validated is `false`.

Reply:
Thanks! You helped to uncover a bigger issue with the image validator which this test should have picked up...